content/content-strategy/manage-content-requests/create-content/why.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: text/textblock
+---
+
+As a government agency, your content is part of your service to the public. It needs to be user focused. When you champion a user approach you will also be able to focus on improving outcomes and measuring success. 
+
+Set expectations for quality content with your authors. Let them know that quality assurance checks and usability testing are part of the content creation process.
+
+Promote the use of the [Content Guide](https://guides.service.gov.au/content-guide/) to learn how to structure content, write in plain English and create accessible content.
```

Comment:
@Phend-117 just noting that this is another reference to `Content guide` (are we going with upper- or lowercase G?) and that the description for what CG is varies.

Reply:
Should be Content Guide - it's the name of a product